RealEstate/apps/core/templates/core/base.html

```diff
@@ -10,7 +10,8 @@
     <![endif]-->
     <meta charset="UTF-8">
     <title>{% block title %}Real Estate{% endblock %}</title>
-
+    <link rel="stylesheet" type="text/css" href="{% static 'style.css' %}" />
+    <link rel="stylesheet" type="text/css" href="{% static 'slide.min.css' %}" />
```

Comment:
I don't think we want to include the slide library on every page. We just need this on the eval and category page. For example, the realtor will never see the slides, but we're requiring him to download the library regardless.